src/cdomains/valueDomain.ml

```diff
@@ -501,8 +501,7 @@ struct
 
 
   let warn_type op x y =
-    if GobConfig.get_bool "dbg.verbose" then
-      ignore @@ printf "warn_type %s: incomparable abstr. values %s and %s at %a: %a and %a\n" op (tag_name (x:t)) (tag_name (y:t)) CilType.Location.pretty !Tracing.current_loc pretty x pretty y
+    Logs.debug "warn_type %s: incomparable abstr. values %s and %s at %a: %a and %a" op (tag_name (x:t)) (tag_name (y:t)) CilType.Location.pretty !Tracing.current_loc pretty x pretty y
```

Comment:
We issue thousands of these for large programs, maybe checking the log level before makes sense?

Reply:
Yes, it is more about the intermediate things being computed before it is determined that no print is necessary because verbose logging is not on.